src/kernel.cpp

```diff
@@ -373,11 +419,39 @@ void init_skgeom_kernel(py::module &m) {
         .def(py::self <= Point_3())
         .def(py::self >= Point_3())
         .def(py::self + Vector_3())
+        //.def(py::self + Point_3())
         .def(py::self - Point_3())
         .def(py::self - Vector_3())
         .def(py::self - CGAL::Origin())
     ;
 
+    py::class_<Line_3>(m, "Line3")
+        .def(py::init<Point_3, Point_3>())
+    	.def(py::init<Point_3, Direction_3>())
```

Comment:
hey, sorry to bother you again but the indentation still does look a little off here.